jestSetup.ts

```diff
@@ -31,4 +32,4 @@ jest.mock('react-native-reanimated', () => {
   return Reanimated
 })
 jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper')
-jest.mock('@emurgo/yoroi-lib-mobile', () => mockYoroiLib)
+jest.mock('@emurgo/cross-csl-mobile', () => mockCSL)
```

Comment:
it doesn't need, since we are mocking `rnhs`.

Reply:
where is that happening?

Reply:
> where is that happening?

inside @__mocks__ (perhaps we should move all inside setup dunno)